jest.config.js

```diff
@@ -0,0 +1,190 @@
+// For a detailed explanation regarding each configuration property, visit:
+// https://jestjs.io/docs/en/configuration.html
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
can we remove all these comments? and keep only one at the very top. It will be way easier to read. It'll be more reliable as well (refer to the docs instead of these comments)

Reply:
I will remove them.